.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "3.1.0"
```

Comment:
Just to double-check, is the idea that we start at a version number after `3.1.0`, the same as the original package when the repo was forked, and then continue with our own versioning?